includes/admin/tracking.php

```diff
@@ -132,20 +136,16 @@ public function send_checkin( $override = false, $ignore_last_checkin = false )
 
 		$this->setup_data();
 
-		$request = wp_remote_post( 'https://easydigitaldownloads.com/?edd_action=checkin', array(
+		wp_remote_post( 'https://easydigitaldownloads.com/?edd_action=checkin', array(
 			'method'      => 'POST',
 			'timeout'     => 20,
```

Comment:
is there a reason the timeout needs to be so high on this? Since not all transport methods support non-blocking requests, we should really lower this to 3 or 5 seconds

Reply:
I think we could, my concern would be that it aborts on the user end if we make it too short, but then it also by default would kill the PHP thread on our end as well.

I'll set it to 8 seconds, which is far less than 20, but in the future we'll get the following into the server end of this telemetry data:
`ignore_user_abort(true);`

With that, at least if the client end aborts, we'll still try and process.

Reply:
Worst case scenario, we miss some telemetry data here and there until then.

Reply:
Right, I think as a policy we should default to not getting data for a week, vs possibly affecting people's sites. I also think eventually EDD will rebuild the server side in non-WordPress and then you don't need to have such a large processing time since you can just get the data and store it incredibly fast.